### ISSUE TITLE
Is there any planned support for Kubernetes 1.24 or 1.25 in the near future?

### ISSUE THREAD

**Issue by lszxyz** [kubeedge/kubeedge#4766]

**Comment by Shelley-BaoYue** [comment]
Yes, we plan to support for Kubernetes 1.24 in the upcoming version. Pr: #4789  It is still in the final debugging phase.

**Comment by yjqg6666** [comment]
The most recent supported version is 1.23 [which is EOL at 2023-02-28](https://kubernetes.io/releases/patch-releases/#non-active-branch-history).

**Comment by yjqg6666** [comment]
Any release schedule for the new version coming?
